lib/sorcery/model.rb

```diff
@@ -140,6 +140,21 @@ def sorcery_config
         self.class.sorcery_config
       end
 
+      #  shows if user is logged in
+      def logged_in?
+        #  returns false if lasts_login is nil
```

Comment:
These comments are not very useful, they don't describe what's going on better than the code itself